frontend/amundsen_application/static/js/components/ResourceListItem/TableListItem/index.tsx

```diff
@@ -34,10 +35,12 @@ const TableListItem: React.FC<TableListItemProps> = ({
   table,
   logging,
   tableHighlights,
+  disabled,
 }) => (
   <li className="list-group-item clickable">
     <Link
       className="resource-list-item table-list-item"
+      style={{ pointerEvents: disabled ? 'none' : '' }}
```

Comment:
We prefer not to use the style prop.
I suggest to conditionally add a class, something like 'is-disabled', and add this styles into the corresponding stylesheet.

Reply:
Thanks for the heads up! I've fixed this.